src/main/java/org/terasology/potions/effect/NoVisibilityEffect.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.potions.effect;
+
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.potions.HerbEffect;
+
+/**
+ * Copyright 2016 MovingBlocks

```

Comment:
Whoops, got a second license header that ended up at the class level javadoc :-)